laboratorios/internal/views/laboratory.py

```diff
@@ -59,8 +59,63 @@ def services_index(request,
             if (employee == service_supervisor):
                 laboratory_services.append(service)
                 break
-    context = {'laboratory_services': laboratory_services, 'laboratory': laboratory,
-               'priorities': all_priorities}
+
+    # get data for track service's graphic
+    month_names = [
+        'Enero',
+        'Febrero',
+        'Marzo',
+        'Abril',
+        'Mayo',
+        'Junio',
+        'Julio',
+        'Agosto',
+        'Septiembre',
+        'Octubre',
+        'Noviembre',
+        'Diciembre'
+    ]
+
+    # get only services of the current laboratory
+    service_request_list = laboratory_services
+
+    my_data = []
+    now = timezone.localtime(timezone.now())
+    for i in range(0, len(service_request_list)):
```

Comment:
El índice iterador (`i`) no se usa en ningún lado, sería mejor usar iteración normal:

    for request in service_request_list:
    ...